core/store/migrate/migrations/0270_log_poller_add_safe_column.sql

```diff
@@ -0,0 +1,10 @@
+-- +goose Up
+-- Add finalized_block_number column to evm.log_poller_blocks
+-- This column is used to track the safe block number for each chain to be consumed from the log poller
+ALTER TABLE evm.log_poller_blocks
+    ADD COLUMN safe_block_number
+        bigint;
```

Comment:
ideally this new column shouldn't support null neither be below 0 (same as `finalized_block_number`): 
```sql
    ADD COLUMN safe_block_number
        bigint not null
        default 0
        check (safe_block_number >= 0); 
```

But I was pointed out by @dhaidashenko that this could take a lot of time to perform this alter table in production.

need to be reviewed later to see what should be done.

Reply:
@mateusz-sekara Do you foresee having issues with an alter table adding a default value to the `evm.log_poller_blocks` due to the high amounts of rows in this migration?

Reply:
ack, will use the same type and constraints for the finalized one (>=0)